src/vis/vis.js

```diff
@@ -120,6 +120,7 @@ var VisModel = Backbone.Model.extend({
 
     this._engine.on(Engine.Events.RELOAD_SUCCESS, this._onEngineReloadSuccess, this);
     this._engine.on(Engine.Events.RELOAD_ERROR, this._onEngineReloadError, this);
+    this._engine.on(Engine.Events.ERROR, this._onEngineError, this);
```

Comment:
What is the difference between `Engine.Events.RELOAD_ERROR` and `Engine.Events.Error`? Its not super clear given that names are a bit similar IMO.

Reply:
Same as above. @IagoLast suggested TILE_ERROR. Good name.